### PR TITLE
Remix projects: don't use colors directly

### PR DESCRIPTION
**Pull request by ruggi**
**Problem:**

The `colors` object is used directly in multiple places inside `style` props, but that _may_ be responsible for the hydration problems we're seeing in production.

**Fix:**

This PR gets rid of that and instead uses `sprinkle` classnames.

(Also reverted to a more sane usage of radix UI's component imports).

### CODE REVIEW

**Review by lankaukk — APPROVED** [review]

**Review by bkrmendy — APPROVED** [review]

File: utopia-remix/app/routes/projects.tsx [hunk 1]
```diff
@@ -551,7 +546,11 @@ const ProjectCard = React.memo(
                     filter: project.deleted === true ? 'grayscale(1)' : undefined,
                   }}
                   title={collaborator.name}
-                  className={sprinkles({ boxShadow: 'shadow' })}
+                  className={sprinkles({
+                    boxShadow: 'shadow',
+                    color: 'white',
+                    background: 'primary',
```
Comment (Rheeseyb) [inline utopia-remix/app/routes/projects.tsx:552]: Should this be `backgroundColor`?

**Review by liady — APPROVED** [review]

File: utopia-remix/app/styles/contextMenu.css.ts [hunk 2]
```diff
@@ -29,10 +29,9 @@ export const contextMenuDropdown = recipe({
     sprinkles({
       borderRadius: 'small',
       background: 'white',
-      color: 'lightModeBlack'
+      color: 'lightModeBlack',
```
Comment (balazsbajorics) [inline utopia-remix/app/styles/contextMenu.css.ts:32]: does it mean we don't yet run eslint / prettier for these files?

Comment (ruggi) [reply utopia-remix/app/styles/contextMenu.css.ts:32]: I think this was untouched after we enabled prettier correctly

File: utopia-remix/app/routes/projects.tsx (hunk 1 above)
Comment (ruggi) [reply utopia-remix/app/routes/projects.tsx]: yeah it should have been

Comment (ruggi) [reply utopia-remix/app/routes/projects.tsx]: [`5999810` (#4936)](https://github.com/concrete-utopia/utopia/pull/4936/commits/59998107213f557c7040bbfb6352208ea8864607)

**Pull request merged**